stable/kommander/values.yaml

```diff
@@ -78,6 +78,15 @@ kubeaddons-catalog:
 grafana:
   enabled: true
 
+  hooks:
+    jobName: grafana-home-dashboard
```

Comment:
Can we make the job name a little clearer? Eg `set-kommander-grafana-home-dashboard` or `set-mgmt-grafana...`. Reason being there is an identical job that runs to set the konvoy home dashboard and this would help to disambiguate on the cluster.

Reply:
The jobs are running in separate namespaces, but I'll change the name here just to make it clearer. Maybe next time I go into prometheus-operator I'll make some improvements there too, that job is currently ambiguously named `post-install-job`